CheckYourEligibility-Admin/Program.cs

```diff
@@ -46,6 +46,16 @@
 
 app.UseRouting();
 app.UseAuthorization();
+app.Use((context, next) =>
+{
+    context.Response.Headers["strict-transport-security"] = "max-age=31536000; includeSubDomains";
```

Comment:
Hey 

I only see a need for the following headers:

strict-transport-security 
content-security-policy
x-frame-options
x-content-type-options

I think you have added the 6 from the API part of the findings on the ITHC doc instead of the Web ones